projects/core/src/icon/icon.stories.ts

```diff
@@ -67,12 +68,102 @@ class AllIcons extends LitElement {
   }
 }
 
+@customElement('demo-icons-collection')
+class CollectionIcons extends LitElement {
+  @state() icons: string[] = [];
+  private collections: any = {};
+
+  private _collection: string;
+  get collection(): string {
+    return this._collection;
+  }
+
+  @property({ type: String })
+  set collection(val: string) {
+    this._collection = val;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    import('@cds/core/icon').then(module => {
+      module.loadChartIconSet();
+      module.loadCommerceIconSet();
+      module.loadCoreIconSet();
+      module.loadEssentialIconSet();
+      module.loadMediaIconSet();
+      module.loadMiniIconSet();
+      module.loadSocialIconSet();
+      module.loadTechnologyIconSet();
+      module.loadTextEditIconSet();
+      module.loadTravelIconSet();
+
+      this.collections = {
+        chart: module.chartCollectionIcons,
+        commerce: module.commerceCollectionIcons,
+        core: module.coreCollectionIcons,
+        essential: module.essentialCollectionIcons,
+        media: module.mediaCollectionIcons,
+        mini: module.miniCollectionIcons,
+        social: module.socialCollectionIcons,
+        technology: module.technologyCollectionIcons,
+        textEdit: module.textEditCollectionIcons,
+        travel: module.travelCollectionIcons,
+      };
+      this.icons = this.collections[this._collection].map((icon: IconShapeTuple) =>
+        ClarityIcons.getIconNameFromShape(icon)
+      );
+    });
+  }
+
+  render() {
+    return html`
+      <strong>${this._collection}</strong>
```

Comment:
Shouldn't this line use the `this.collections` to go through the get accessor? It doesn't matter now, but if the getter is changed...

Reply:
Not likely to change, but I don't mind updating it. 
This code is quite storybook oriented. I was thinking that I can extract some parts of it in a common util, but with the current code structure and lack of prioritization for Core tasks, it may take me more time than I want to invest right now.